packages/kobotoolbox/test/index.js

```diff
@@ -1,13 +1,28 @@
 import chai from 'chai';
 const { expect } = chai;
+import { enableMockClient } from '@openfn/language-common/util';
 
 import nock from 'nock';
 
-import Adaptor from '../src';
+import Adaptor, { http } from '../src';
+
 const { execute } = Adaptor;
 
 import { getSubmissions, getForms, getDeploymentInfo } from '../src/Adaptor';
 
+const testServer = enableMockClient('https://test.kobotoolbox.org');
+const jsonHeaders = {
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
+const configuration = {
+  username: 'test',
+  password: 'strongpassword',
+  apiVersion: 'v2',
+  baseURL: 'https://test.kobotoolbox.org',
```

Comment:
Oh, interesting that this is `baseURL` and not `baseUrl`. This comment is a reminder to me to maybe change this